app/models/financialDetails/ChargeItem.scala

```diff
@@ -125,6 +125,12 @@ case class ChargeItem (
 
   val isReviewAndReconcileCharge: Boolean = isPoaReconciliationCredit || isPoaReconciliationDebit
 
+  val isPenalty: Boolean = transactionType == LateSubmissionPenalty ||
```

Comment:
if we can refactor this a bit and create a separate list for penalties ?
```
 val penalties = List(LateSubmissionPenalty, FirstLatePaymentPenalty, SecondLatePaymentPenalty)
penalties.contains(_)
```

Reply:
we might need to introduce a new field in ChargeItem, ie category or similar, might be worth to create a tech debt to refactor this in the future;
there are two candidates so far: POA, Penalties, None 
              
<img width="1130" alt="image" src="https://github.com/user-attachments/assets/ff856470-9543-47ec-8214-b067a78e58e3" />